src/Avalonia.Controls/ExperimentalAcrylicBorder.cs

```diff
@@ -0,0 +1,114 @@
+﻿using Avalonia.Controls.Utils;
+using Avalonia.Layout;
+using Avalonia.Media;
+using Avalonia.Platform;
+using System;
+
+namespace Avalonia.Controls
+{
+    public class ExperimentalAcrylicBorder : Decorator
+    {
+        public static readonly StyledProperty<CornerRadius> CornerRadiusProperty =
+            Border.CornerRadiusProperty.AddOwner<ExperimentalAcrylicBorder>();
+
+        public static readonly StyledProperty<ExperimentalAcrylicMaterial> MaterialProperty =
+            AvaloniaProperty.Register<ExperimentalAcrylicBorder, ExperimentalAcrylicMaterial>(nameof(Material));
+
+        private readonly BorderRenderHelper _borderRenderHelper = new BorderRenderHelper();
+
+        static ExperimentalAcrylicBorder()
+        {
+            AffectsRender<ExperimentalAcrylicBorder>(
+                MaterialProperty,
+                CornerRadiusProperty);
+        }
+
+
+        /// <summary>
+        /// Gets or sets the radius of the border rounded corners.
+        /// </summary>
+        public CornerRadius CornerRadius
+        {
+            get { return GetValue(CornerRadiusProperty); }
+            set { SetValue(CornerRadiusProperty, value); }
+        }
+
+        public ExperimentalAcrylicMaterial Material
+        {
+            get => GetValue(MaterialProperty);
+            set => SetValue(MaterialProperty, value);
+        }
+
+        protected override void OnAttachedToVisualTree(VisualTreeAttachmentEventArgs e)
+        {
+            base.OnAttachedToVisualTree(e);
+
+            var tl = (e.Root as TopLevel);
+
+            tl.GetObservable(TopLevel.ActualTransparencyLevelProperty)
```

Comment:
You need to dispose that subscription on detachment

Reply:
done.